presto-docs/src/main/sphinx/release/release-0.152.1-t.1.rst

```diff
@@ -4,30 +4,162 @@ Release 0.152.1-t.1
 
 Presto 0.152.1-t.1 is equivalent to Presto release 0.152.1, with some additional features.
 
-Prepared Statements
--------------------
-Add support for Prepared statements and parameters via sql syntax.
+Security
+--------
+LDAP Authentication
 
-    * PREPARE
-    * DEALLOCATE PREPARE
-    * EXECUTE
-    * DESCRIBE INPUT
-    * DESCRIBE OUTPUT
+SQL
+---
+Additional correlated Scalar Subqueries
+Additional prepared statement syntax: DESCRIBE INPUT and DESCRIBE OUTPUT
 
 Data Types
 ----------
-Add FLOAT support to the Hive connector
-Add Char support to Hive connector
-Additional Varchar(x) function implementations
+Additional Varchar(x) and Char(x) function implementations
 Additional Decimal functions implementations
 
 Documentation
 -------------
-Additional Kerberos
-Grant/Revoke
-Presto-Admin
-Presto YARN Integration
-Presto Ambari Integration
-TINYINT, SMALLINT, INTEGER
+Installation of Presto via Presto Admin
+Presto Tuning Guide
+
+Decimal
+-------
+Faster Decimal implementation
+
+
+Join Filters
+------------
+Move certain filters in the WHERE clauses to be executed as part of the INNER JOIN rather than a filter after the join
+For example: ``SELECT * FROM t t1 JOIN t t2 ON t1.a = t2.a WHERE (t1.b+t2.b)*5 > 100000000``
+Code generation for joins with filters
```

Comment:
@kokosing -- should we add something here about subqueries? e.g. things that are in 152.1-t but not in 152.